languages/indonesian/pack/src/main/res/xml/indonesian_keyboards.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Keyboards>
+    <!-- Dictionary attributes:
+         "nameResId" : the name of the keyboard. Should be short. Reference to a string in values.xml
+         "defaultDictionaryLocale" : the ID of the keyboard's (ISO) locale, e.g., "en", "iw", "ar", "ru", etc. The dictionary should declare the same locale.
+         "id" : UUID for this keyboard. Obtain from here: http://www.famkruithof.net/uuid/uuidgen
+         "iconResId" : a reference to a drawable which can be used to show a flag in the notification bar.
+         "layoutResId" : a reference to a keyboard layout XML. See res/xml/qwerty.xml
+    -->
+    <Keyboard nameResId="@string/indonesian_keyboard" iconResId="@drawable/ic_status_indonesian"
+              layoutResId="@xml/qwerty" id="201208c2-b269-47f8-a8cc-e6dc39f0148a"
+              defaultDictionaryLocale="id" description="@string/indonesian_keyboard_description"
+              index="1" defaultEnabled="true"
+              physicalKeyboardMappingResId="@xml/english_physical" />

```

Comment:
If you want to have a physical-translation, create a new one for it with `indonesian_` prefi

Reply:
I've changed this. This is what I asked in issue. We need a description of "physicalKeyboardMappingResId" and other value. Like old file had.